extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -836,22 +806,52 @@ class CornerstoneViewportService extends PubSubService implements IViewportServi
       return { properties, volumeId };
     });
 
+    // For SEG and RT viewports
+    this._processOverlaysForViewport(viewport);
+
+    // Always load the reference volume if the displaySet is an overlay
```

Comment:
If this is still true, then do you need to do something here?  If not, can you remove